tests/github-action-helper.sh

```diff
@@ -31,9 +31,21 @@ deploy_rook() {
   kubectl create -f https://raw.githubusercontent.com/rook/rook/master/deploy/examples/filesystem-test.yaml
   kubectl create -f https://raw.githubusercontent.com/rook/rook/master/deploy/examples/subvolumegroup.yaml
   kubectl create -f https://raw.githubusercontent.com/rook/rook/master/deploy/examples/toolbox.yaml
+  kubectl create -f https://raw.githubusercontent.com/rook/rook/master/deploy/examples/csi/cephfs/storageclass.yaml
   kubectl create -f https://raw.githubusercontent.com/rook/rook/master/deploy/examples/csi/rbd/storageclass-test.yaml
+  curl https://raw.githubusercontent.com/rook/rook/master/deploy/examples/csi/cephfs/storageclass.yaml -o storageclass.yaml
+  sed -i "s|name: rook-cephfs|name: rook-cpehfs-retain|g" storageclass.yaml
+  sed -i "s|reclaimPolicy: Delete|reclaimPolicy: Retain|g" storageclass.yaml
+  kubectl create -f storageclass.yaml
   kubectl create -f https://raw.githubusercontent.com/rook/rook/master/deploy/examples/csi/rbd/pvc.yaml
   kubectl create -f https://raw.githubusercontent.com/rook/rook/master/deploy/examples/csi/cephfs/pvc.yaml
+  curl https://raw.githubusercontent.com/rook/rook/master/deploy/examples/csi/cephfs/pvc.yaml -o pvc.yaml
+  sed -i "s|name: cephfs-pvc|name: cephfs-pvc-retain|g" pvc.yaml
+  sed -i "s|storageClassName: rook-cephfs|storageClassName: rook-cephfs-retain|g" pvc.yaml
+  kubectl create -f pvc.yaml
+  kubectl delete pvc cephfs-pvc-retain
+  "${pvname:=$(kubectl get pv | grep cephfs-pvc-retain | awk '{print $1}')}"
```

Comment:
```suggestion
  PVNAME:=$(kubectl get pv | grep cephfs-pvc-retain | awk '{print $1}')
```
or 
```
:  "${PVNAME:=$(kubectl get pv | grep cephfs-pvc-retain | awk '{print $1}')}"
```

Reply:
Initially I tried with the first suggestion, but even that is failing, will go for the next.